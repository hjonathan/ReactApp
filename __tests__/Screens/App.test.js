import React from "react";
import toJson from "enzyme-to-json";
import AppFn from "../../App";

describe("Component >>> App", () => {
    let App;

    it("Render App", () => {
        expect(toJson(AppFn)).toMatchSnapshot();
    });
});
