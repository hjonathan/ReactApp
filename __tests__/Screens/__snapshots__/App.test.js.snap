// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component >>> App Render App 1`] = `undefined`;
